LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_LED" Id="{5fae4b24-b1d6-4ac5-a010-9aff4e2893db}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_LED
+(*
+    Reads a percentage of illumination determined by the user and converts it to a raw value
+    to send to the terminal output via the FB_AnalogOutput.
+	
+	The determination of the full scale raw value (FSV) is calculated via the iTermBits,
+    which is setup as an input variable to be set according to the situation. This value
+    is bounded by the FSV of the terminal set by the Beckhoff architecture: (2^15 - 1) = 32767.  
+	
+	Through the configurable max raw value sent to the terminal output, this LED function block
+    is applicable to a wider variety of hardware: whether it be simple Analog Voltage control,
+    PWM voltage output, or current controlled output. 
+
+    As an initial application, this FB will be instituted for systems of vaccum chamber LED 
+    illuminators on TMO endstations, which are rated for 12 V max. They will be dimmed via the
+    EL2502 terminal 24 V PWM output voltage control from 0-50% duty cycle.
+	
+    2022-5-03 Maarten Thomas-Bosum
+*)
+VAR_INPUT
+	iTermBits : UINT;
```

Comment:
`fIlluminatorPercent` should be an input, should it not?

Reply:
I tested the code as is by creating a test IOC, and I was able to caget and caput the illuminator percentage fine from EPICS. 

I can try to make fIlluminatorPercent an input variable and see how it functions.

Reply:
As an input, you have the option of controlling it from the PLC or from EPICS.

It seems unnecessarily restrictive to say "this is meant only for EPICS".

Thoughts on the PV name there as @ZLLentz had pointed out [above](https://github.com/pcdshub/lcls-twincat-general/pull/63#discussion_r871547516), @Mbosum?

Reply:
@ZLLentz I'm not convinced we need bLedPower to be controllable from EPICS. I think I at least initially just wanted it to be there as a sanity check when the user sets the illumination percentage.

Reply:
> I'm not convinced we  we need bLedPower to be controllable from EPICS

As written, it's _only_ controllable from EPICS.

Reply:
I changed it to an input variable now